unison-cli/src/Unison/Codebase/Editor/Command.hs

```diff
@@ -273,55 +273,55 @@ lookupEvalResult v (_, m) = view _5 <$> Map.lookup v m
 
 commandName :: Command m i v a -> String
 commandName = \case
-  Eval{}                      -> "Eval"
-  UI                          -> "UI"
-  DocsToHtml{}                -> "DocsToHtml"
-  ConfigLookup{}              -> "ConfigLookup"
-  Input                       -> "Input"
-  Notify{}                    -> "Notify"
-  NotifyNumbered{}            -> "NotifyNumbered"
-  AddDefsToCodebase{}         -> "AddDefsToCodebase"
-  CodebaseHashLength          -> "CodebaseHashLength"
-  TypeReferencesByShortHash{} -> "TypeReferencesByShortHash"
-  TermReferencesByShortHash{} -> "TermReferencesByShortHash"
-  TermReferentsByShortHash{}  -> "TermReferentsByShortHash"
-  BranchHashLength            -> "BranchHashLength"
-  BranchHashesByPrefix{}      -> "BranchHashesByPrefix"
-  ParseType{}                 -> "ParseType"
-  LoadSource{}                -> "LoadSource"
-  Typecheck{}                 -> "Typecheck"
-  TypecheckFile{}             -> "TypecheckFile"
-  Evaluate{}                  -> "Evaluate"
-  Evaluate1{}                 -> "Evaluate1"
-  PutWatch{}                  -> "PutWatch"
-  LoadWatches{}               -> "LoadWatches"
-  LoadLocalRootBranch         -> "LoadLocalRootBranch"
-  LoadLocalBranch{}           -> "LoadLocalBranch"
-  Merge{}                     -> "Merge"
-  ViewRemoteBranch{}          -> "ViewRemoteBranch"
-  ImportRemoteBranch{}        -> "ImportRemoteBranch"
-  SyncLocalRootBranch{}       -> "SyncLocalRootBranch"
-  SyncRemoteRootBranch{}      -> "SyncRemoteRootBranch"
-  AppendToReflog{}            -> "AppendToReflog"
-  LoadReflog                  -> "LoadReflog"
-  LoadTerm{}                  -> "LoadTerm"
-  LoadType{}                  -> "LoadType"
-  LoadTypeOfTerm{}            -> "LoadTypeOfTerm"
-  PutTerm{}                   -> "PutTerm"
-  PutDecl{}                   -> "PutDecl"
-  IsTerm{}                    -> "IsTerm"
-  IsType{}                    -> "IsType"
-  GetDependents{}             -> "GetDependents"
-  GetTermsOfType{}            -> "GetTermsOfType"
-  GetTermsMentioningType{}    -> "GetTermsMentioningType"
-  Execute{}                   -> "Execute"
-  CreateAuthorInfo{}          -> "CreateAuthorInfo"
-  RuntimeMain                 -> "RuntimeMain"
-  RuntimeTest                 -> "RuntimeTest"
-  HQNameQuery{}               -> "HQNameQuery"
-  LoadSearchResults{}         -> "LoadSearchResults"
-  GetDefinitionsBySuffixes{}  -> "GetDefinitionsBySuffixes"
-  FindShallow{}               -> "FindShallow"
-  ClearWatchCache{}           -> "ClearWatchCache"
-  MakeStandalone{}            -> "MakeStandalone"
-  FuzzySelect{}               -> "FuzzySelect"
+  Eval {} -> "Eval"
```

Comment:
I ormolu'd this.

It may cause a merge conflict this time, but at least shouldn't cause conflicts in the future from like the previous version did every time you added a new longest constructor.